examples/dataprofiler_expectations/README.md

```diff
@@ -0,0 +1,13 @@
+## Data Profiler + Great Expectation
+[Great Expectations](https://greatexpectations.io/) is an open source tool built around the idea that should always know what to expect from your data. It helps data teams eliminate pipeline debt, through data testing, documentation, and profiling.
+
+See a full list of Data Profiler Expectations available through Great Expectations on the [package page](https://greatexpectations.io/packages/capitalone_dataprofiler_expectations). 
+### Local Setup
+1. Navigate to the GE examples directory: `cd examples/great_expectations/`
+2. Initialize your virtual environment: `python3 -m venv venv`
+3. Activate the virtual environment: `source venv/bin/activate`
+4. Download and install Capital One's DataProfiler Expectations
+   1. Clone the great expectations: `git clone https://github.com/great-expectations/great_expectations.git`
+   2. Add `py_modules=[]` to `great_expectations/contrib/capitalone_dataprofiler_expectations/setup.py` under `setuptools.setup`
+   3. Install the package to virtual environment: `pip install -e  great_expectations/contrib/capitalone_dataprofiler_expectations`
```

Comment:
Could we do a pip install pointing to the repo instead of cloning it?

Reply:
https://stackoverflow.com/questions/13566200/how-can-i-install-from-a-git-subdirectory-with-pip

Reply:
Would have to validate if this works for custom expectations.... 


I think the below command will work if we update the [`setup.py` in `capitalone_dataprofiler_expectations`](https://github.com/pypa/setuptools/issues/3197#issuecomment-1078770109) on the GE side of the house: 
```
pip install git+https://github.com/great-expectations/great_expectations.git@develop#subdirectory=contrib/capitalone_dataprofiler_expectations/
```

Reply:
Would have to validate if this works for custom expectations.... 


I think the below command will work if we update the [`setup.py` in `capitalone_dataprofiler_expectations`](https://github.com/pypa/setuptools/issues/3197#issuecomment-1078770109) on the GE side of the house: 
```
pip install git+https://github.com/great-expectations/great_expectations.git@develop#subdirectory=contrib/capitalone_dataprofiler_expectations/
```

Reply:
I updated the readme per the suggestion and everything installed correctly, the example ran successfully.